src/lib.rs

```diff
@@ -387,11 +387,7 @@ macro_rules! impl_ranged {
             type Error = TryFromIntError;
 
             fn try_from(value: $type<MIN, MAX>) -> Result<Self, Self::Error> {
-                if (MIN..=MAX).contains(&value.0) {
```

Comment:
I suspect that we don't need to check the range, because `value` is already of type like `U32<_. _>`, in which the assertion is already done.  I simply removed it.

Reply:
Heh, you're 100% right. Incidentally I had this _exact_ change locally, but for unknown reasons I left the repo in a dirty state and never committed it.